dev-docs/RFCs/v5.3/attribute-buffer-rfc.md

```diff
@@ -0,0 +1,36 @@
+# RFC: Attribute Buffer Handling
+
+* **Authors**: Xiaoji Chen
+* **Date**: Mar 21, 2018
+* **Status**: **Pending Review**
+
+Related discussions:
+[#1527](https://github.com/uber/deck.gl/pull/1527)
+
```

Comment:
Good to have a `overview` section with 2 line statement of what is being proposed. Something like: "This RFC proposed changes to AttributeManager and a new Attribute class to enable control of GPU buffer lifecycles, leading to reduced memory usage."